javascript-practice/src/js/templates/userTemplate.js

```diff
@@ -1,6 +1,7 @@
 import editIcon from '../../assets/icons/arrow-circle-down.svg';
 import deleteIcon from '../../assets/icons/arrow-circle-down-copy.svg';
-import { MONTH_NAMES, PRIMARY_ROLES } from '../constants/usersTable';
+import { MONTH_NAMES} from '../constants/month';
+import { PRIMARY_ROLES, ROLES } from '../constants/role';
```

Comment:
Could you check logic code?

Reply:
I don't use it here, I removed it, thanks